recipes/seasocks/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "seasocks" %}
+{% set version = "1.4.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/mattgodbolt/seasocks/archive/v{{ version }}.tar.gz
+  sha256: 82211959cf8aabc85b300358c5f68cf9c56dfdf4eaa09e548580fce908acfc1b
+
+build:
+  number: 0
+  skip: True  # [osx]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - ninja
+    - python
+
+  host:
+    - zlib
+
+test:
+  requires:
+    - cmake
+    - {{ compiler('cxx') }}
+    - ninja
+  files:
+    - test
+
+  commands:
```

Comment:
Seems like this produces a shared library, please test using `test -f` for the existence of the correct file name